### ISSUE TITLE
Garbage code in markdown - automatic way to fix?

### ISSUE THREAD

**Issue by librarychik** [librarychik/eBranch-documentation#1]
The transformed markdown document has a lot of garbage code, like carets and extra spaces.  I will look around to see if there is any other automatic tool to finish cleaning it up, but thought @chrisdaaz might have some ideas? :)

It could be it is a manual job the rest of the way.

**Comment by chrisdaaz** [comment]
Yes, I've seen this before in conversions from DOCX to Markdown. You might have some luck using the Find and Replace function in your text editor, where you find the caret `>` and replace it with nothing. That would be my first step.

More advanced find and replace might involve [Regular Expressions](https://dev.to/rfornal/vs-code-search-and-replace-regex-mn2) or the [`sed`](https://www.geeksforgeeks.org/sed-command-in-linux-unix-with-examples/) unix utility (if you're on macOS or Linux).
